frontend/src/utils/randomDateBetweenCurrentDayAnd21DaysAgo.js

```diff
@@ -0,0 +1,17 @@
+export function randomDateBetweenCurrentDayAnd21DaysAgo() {
```

Comment:
This is defined in a ery funny place. It's only used in the story, right? Maybe it should be with the story.

That said, maybe it shouldn't exist at all. Stories should be pretty predictable. 

Consider using a generator to yield a non-random sequence of dates.